packages/core/src/store/onBoardingSlice.ts

```diff
@@ -0,0 +1,21 @@
+import { StateCreator } from 'zustand'
```

Comment:
I think it's a bit overkill bringing the onboarding logic in Zustand as well —

IMO the less information inside of Zustand the better — if it can be covered by a simple `useState`, it should be preferably done via a `useState` !

Probably the only data that needs to be in here is the design token data!

Reply:
I have removed onboarding from zustand. I agree it was kind of overkill.